src/ui/components/ExperimentBrowserPage.tsx

```diff
@@ -0,0 +1,114 @@
+import { FC, useCallback, useEffect, useState } from "react";
+import { NimbusExperiment } from "@mozilla/nimbus-schemas";
+
+const PROD_URL =
+  "https://experimenter.services.mozilla.com/api/v6/experiments/";
+const STAGE_URL =
+  "https://stage.experimenter.nonprod.dataops.mozgcp.net/api/v6/experiments/";
+
+type Status = "Live" | "Preview";
+
+enum Environment {
+  Prod = "prod",
+  Stage = "stage",
+}
+
+const ExperimentBrowserPage: FC = () => {
+  const [environment, setEnvironment] = useState<Environment>(Environment.Prod);
+  const [status, setStatus] = useState<Status>("Live");
+  const [experiments, setExperiments] = useState<NimbusExperiment[]>([]);
+
+  const fetchExperiments = useCallback(
+    async (forceRefresh = false) => {
+      let url = environment === Environment.Prod ? PROD_URL : STAGE_URL;
+      url += `?status=${status}`;
+      if (forceRefresh) {
+        url += `&bust-cache=${Date.now()}`;
+      }
+
+      try {
+        const fetchedExperiments = await fetch(url).then(
+          (rsp) => rsp.json() as Promise<NimbusExperiment[]>,
+        );
+        setExperiments(fetchedExperiments);
+      } catch (error) {
+        console.error("Error fetching experiments:", error);
+      }
+    },
+    [environment, status],
+  );
+
+  useEffect(() => {
+    const fetchData = async () => {
+      await fetchExperiments();
+    };
+    void fetchData();
+  }, [environment, status, fetchExperiments]);
+
+  return (
+    <div className="experiment-viewer">
+      <h1 className="experiment-viewer__title">{status} Experiments</h1>
+      <div className="experiment-viewer__controls">
+        <label className="experiment-viewer__controls__title">
+          Environment:
+          <select
+            className="experiment-viewer__controls__dropdown"
+            value={environment}
+            onChange={(e) => setEnvironment(e.target.value as Environment)}
+          >
+            <option value="prod">Prod</option>
+            <option value="stage">Stage</option>
+          </select>
+        </label>
+        <label className="experiment-viewer__controls__title">
```

Comment:
BEM doesnt allow nesting multiple elements in one selector. See [the section in the FAQ](https://getbem.com/faq/#css-nested-elements).

I think I would change `.experiment-viewer__controls__title` to be `.experiment-viewer-controls__title`